### PR TITLE
Automatic parent element lookup fix

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -134,6 +134,31 @@ Basic usage
     a_view.an_input.is_displayed
 
 
+``__locator__()`` and ``__element__()`` protocol
+------------------------------------------------
+
+To ensure good structure, a protocol of two methods was introduced. Let's talk a bit about them.
+
+``__locator__()`` method is not implemented by default on ``Widget`` class. Its sole purpose is to
```
Comment (izapolsk) [inline README.rst]: "not implemented" ? typo

Comment (mfalesni) [inline README.rst]: It is indeed not. Widget does not contains `__locator__` by default. You either specify ROOT or define it yourself. :smiley:

Comment (izapolsk) [inline README.rst]: but it gets generated in meta class

Comment (mfalesni) [inline README.rst]: Only if you pass it ROOT, then it is generated in the meta class. Otherwise you can have a widget that does not have a `__locator__`

Comment (izapolsk) [inline README.rst]: oh, yes, you're right I didn't notice that above if

**Pull request merged**